nav2_costmap_2d/test/integration/footprint_tests.cpp

```diff
@@ -143,40 +169,25 @@ TEST( Costmap2DROS, footprint_from_same_level_param )
   EXPECT_EQ( 0.0f, footprint[ 2 ].z );
 }
 
-TEST( Costmap2DROS, footprint_from_xmlrpc_param_failure )
-{
-  ASSERT_ANY_THROW( Costmap2DROS cm( "xmlrpc_fail", *tf_ ));
-}
-
-TEST( Costmap2DROS, footprint_empty )
-{
-  Costmap2DROS cm( "empty", *tf_ );
-  std::vector<geometry_msgs::Point> footprint = cm.getRobotFootprint();
-  // With no specification of footprint or radius, defaults to 0.46 meter radius plus 0.01 meter padding.
-  EXPECT_EQ( 16, footprint.size() );
-
-  EXPECT_NEAR( 0.47f, footprint[ 0 ].x, 0.0001 );
-  EXPECT_NEAR( 0.0f, footprint[ 0 ].y, 0.0001 );
-  EXPECT_EQ( 0.0f, footprint[ 0 ].z );
-}
-
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "footprint_tests_node");
+  rclcpp::init(argc, argv);
+  auto node = rclcpp::Node::make_shared("footprint_tests");
 
-  tf_ = new tf2_ros::Buffer( ros::Duration( 10 ));
+  tf_ = new tf2_ros::Buffer(node->get_clock(), tf2::durationFromSec(10));
```

Comment:
You could leave out the durationFromSec(10) bit. That is the default value for the Buffer constructor.